frontend/console/src/features/graph/graph-utils.ts

```diff
@@ -73,17 +73,17 @@ const createModuleChildren = (module: Module, nodePositions: Record<string, { x:
       createChildNode(module.name, nodeId(module.name, secret.secret?.name), secret.secret?.name || '', 'secret', nodePositions, secret, isDarkMode),
     ),
     // Create nodes for subscriptions
-    ...(module.subscriptions || []).map((subscription: Subscription) =>
-      createChildNode(
-        module.name,
-        nodeId(module.name, subscription.subscription?.name),
-        subscription.subscription?.name || '',
-        'subscription',
-        nodePositions,
-        subscription,
-        isDarkMode,
-      ),
-    ),
+    // ...(module.subscriptions || []).map((subscription: Subscription) =>
```

Comment:
@wesbillman This PR removes Subscriptions as a standalone concept in the schema. And I'm not sure how we want that to affect the console... I've just commented out the broken parts for now